doc/classes/InstancePlaceholder.xml

```diff
@@ -33,6 +27,7 @@
 			<argument index="0" name="with_order" type="bool" default="false">
 			</argument>
 			<description>
+				Returns the changed properties and their values as key/value pairs. If [with_order] is true, the dictionary will have an array with the property names in the order of modification under the key [code].order[/code].
```

Comment:
Should be `[code]with_order[/code]`.
Does the key name really include a dot or did you want to signify how it's accessed? Since you mentioned the key, I would just put the key name, without dot.

Reply:
Seems to be dictionary[".order"]:
https://github.com/godotengine/godot/blob/e61edd69ea7a447242736e42462bcc4850339c5f/scene/main/instance_placeholder.cpp#L132